dist/index.html

```diff
@@ -9,11 +9,11 @@
 </head>
 <body>
     <nav class="nav-bar">
-        <img class="nav-logo pointer" id="logo" src="./images/home-button.png" alt="Take Me Home">
+        <img class="nav-logo pointer" id="logo" src="./images/home-button.png" alt="Take Me Home" tabindex="0">
```

Comment:
Love the tab-index. So much better than re-creating everything as a button!

Reply:
+1 - great fix here and a huge learning!